geemap/map_widgets.py

```diff
@@ -822,11 +823,95 @@ def _render_layer_row(self, layer):
         )
         settings_button.on_click(self._on_layer_settings_click)
 
+        spinner = ipywidgets.Button(
```

Comment:
Instead of a standalone spinner, could we combine the spinner and delete button? It saves valuable screen real estate. We can use `ipyevents.Event` for mouse events.

https://github.com/gee-community/geemap/assets/12777663/a39ff242-80a3-4ef5-a8a2-70dd38e93fb5

Reply:
Good suggestion. I can combine the spinner and delete button as a single button with ipyevents. I am not sure if it is feasible to display the confirmation widget on top of the layer manager just like what you what in the demo.

Reply:
See the new demo
https://github.com/gee-community/geemap/assets/5016453/49d48b31-884a-40ec-9b06-dad40d6f130b